.gitignore

```diff
@@ -21,3 +21,4 @@
 
 # virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
 hs_err_pid*
+/target/
```

Comment:
This is part of your other PR, could you remove it from here?

Reply:
Deleted. I also removed it from the other PR as it was unrelated. Sorry about that.